tofu/geom/_core.py

```diff
@@ -3862,6 +3862,9 @@ def from_svg(
         cls,
         pfe,
         res=None,
+        point1=None,
+        point2=None,
+        length=None,
```

Comment:
tbh, I feel like `point1` and `point2` are not specific enough names... `point_ref1/2`? `strart_ref/end_ref`? or even `reference1/2`? what do you think?

Reply:
done